cmd/ci-reporter/report.go

```diff
@@ -38,167 +38,111 @@ type requiredJob struct {
 }
 
 func main() {
-	boolPtr := flag.Bool("short", false, "a short report for mails and slack")
+	// parse flags
+	isShortFlagSet := flag.Bool("short", false, "A short report for mails and slack")
```

Comment:
naming the variable `short` is more canonical IMHO
```suggestion
	short := flag.Bool("short", false, "A short report for mails and slack")
```